intezer_sdk/alerts.py

```diff
@@ -20,3 +24,133 @@ def get_alerts_by_alert_ids(alert_ids: List[str],
     api = IntezerApi(api or get_global_api())
     result = api.get_alerts_by_alert_ids(alert_ids, environments)
     return result['alerts_count'], result['alerts']
+
+
+class Alert:
+    """
+    The Alert class is used to represent an alert from the Intezer Analyze API.
+
+    :ivar alert_id: The alert id.
+    :vartype alert_id: str
+    :ivar raw_alert: The raw alert data.
+    :vartype raw_alert: dict
+    :ivar source: The source of the alert.
+    :vartype source: str
+    :ivar verdict: The verdict of the alert.
+    :vartype verdict: str
+    :ivar family_name: The family name of the alert.
+    :vartype family_name: str
+    :ivar sender: The sender of the alert.
+    :vartype sender: str
+    :ivar url: URL for the alert in Intezer's website.
+    :vartype url: str
+    """
+    def __init__(self, alert_id: str, api: IntezerApiClient = None):
+        """
+        Create a new Alert instance with the given alert id.
+        Please note that this does not query the Intezer Analyze API for the alert data, but rather creates an Alert
+        instance with the given alert id.
+
+        :param alert_id: The alert id.
+        :param api: The API connection to Intezer.
+        """
+        self.alert_id: str = alert_id
+        self._api = IntezerApi(api or get_global_api())
+        self.raw_alert: Optional[Dict] = None
+        self.source: Optional[str] = None
+        self.verdict: Optional[str] = None
+        self.family_name: Optional[str] = None
+        self.sender: Optional[str] = None
+        self.url: Optional[str] = None
+
+    def refresh_alert(self):
```

Comment:
I think it's better to use the same mechanism of the analysis - "send", "check_status" and "result" to be consistent